bundle/package.json

```diff
@@ -65,6 +65,7 @@
     "@pixi/filter-radial-blur": "8.0.0",
     "@pixi/filter-reflection": "8.0.0",
     "@pixi/filter-rgb-split": "8.0.0",
+    "@pixi/filter-shockwave": "8.0.0",
```

Comment:
Why do we need to remove shockwave?

Reply:
Hey Matt, we previously removed it due to it already existed in the older v8 rc, but has now moved back to this repo 👍